chainntnfs/dcrdnotify/dcrd.go

```diff
@@ -5,16 +5,17 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/decred/dcrd/txscript/v4/stdaddr"
```

Comment:
may want to group this with the non-stdlib imports below

Reply:
Thank you for the review! These were all added automatically via some scripts I wrote to ease the conversion process, that's why there were ungrouped.